src/index.js

```diff
@@ -0,0 +1,169 @@
+'use strict';
+
+var Image = require('image-js');
+
+const defaultOptions = {
+    lowThreshold: 10,
+    highThreshold: 30,
+    blur: 1.1,
+    brightness: 255
```

Comment:
when you refer that those value should be calculated in the function you refer to the low and high threshold, because also in opencv they have those ones as a parameters 

http://docs.opencv.org/2.4/modules/imgproc/doc/feature_detection.html?highlight=canny#canny

I can remove the blur and put the brightness as you said, for those ones there is no problem